projects/ngx-openlayers/src/lib/interactions/dragbox.component.ts

```diff
@@ -18,6 +17,8 @@ export class DragBoxInteractionComponent implements OnInit, OnDestroy {
   @Input()
   boxEndCondition: EndCondition;
 
+  onBoxEnd = undefined;
```

Comment:
According to `5.3.3` docs `onBoxEnd` is not optional (https://openlayers.org/en/v5.3.0/apidoc/module-ol_interaction_DragBox.html), but this is actually a typo in the documentation (https://github.com/openlayers/openlayers/pull/9042). Unfortunately the typings on DefinitelyTyped also have this typo.